ext/drreg/drreg.c

```diff
@@ -79,6 +79,14 @@
  */
 #define AFLAGS_ALIAS_REG (DR_REG_STOP_GPR + 1)
 
+/* Used to indicate that "no offset" is being used. */
```

Comment:
Used where?  Only in gpr_to_non_drreg_slot_offs[]?  Move the def down next to that var?